src/lib.rs

```diff
@@ -112,6 +116,18 @@ impl Program {
                         return Ok(locals.remove(a as usize));
                     }
                 }
+                Jump(a) => {
+                    iptr += a as isize;
+                    continue;
+                }
+                CondJump(a, b, c) => {
+                    if locals[a as usize] != I(0) {
```

Comment:
We should consider `B(false)` to be false.
If we want to be lenient, we should also consider both `I(0)` and `F(0.0)` as well.
If we don't want to be as lenient (I'm leaning towards this), we might want to throw an error if we get a non-boolean value here.